docs/content/direct/examples.md

```diff
@@ -60,20 +60,14 @@ manager which connects to the `wds1` front-end and the `imbs1` OCM control plane
    kflex create wds1 -p kubestellar
    ```
 
-1. Run the OCM based transport controller as executable process.  
-**NOTE**: This is work in progress, in the future the controller will be deployed through a Pod and optionally a Helm chart.
-
-   OCM based transport controller is built in a different package, so we need to download the package and run the executable as background process.
+1. Run the OCM based transport controller in a pod.  
+**NOTE**: This is work in progress, in the future the controller will be deployed through a Helm chart.
 
+   Run [transport deployment script](../../../hack/deploy-transport-controller.sh).  
+   This script expects to get three arguments - (1) wds name; (2) imbs name; and (3) transport controller image.  
+   For example, one can deploy transport controller using the following command:
    ```shell
-   OS=$(go env GOOS)
-   ARCH=$(go env GOARCH)
-   OCM_TRANSPORT_PLUGIN_RELEASE="0.1.0-rc2"
-   OCM_TRANSPORT_PLUGIN_TAR=ocm-transport-plugin_${OCM_TRANSPORT_PLUGIN_RELEASE}_${OS}_${ARCH}.tar.gz
-   wget https://github.com/kubestellar/ocm-transport-plugin/releases/download/v${OCM_TRANSPORT_PLUGIN_RELEASE}/${OCM_TRANSPORT_PLUGIN_TAR}
-   tar -xvf ${OCM_TRANSPORT_PLUGIN_TAR} transport-controller
-   rm -fr bin ${OCM_TRANSPORT_PLUGIN_TAR}
-   ./transport-controller --transport-context imbs1 --wds-context wds1 --wds-name wds1 &> transport.log &
+   ./hack/deploy-transport-controller.sh wds1 imbs1 ghcr.io/kubestellar/ocm-transport-plugin/transport-controller:0.1.0-rc4
```

Comment:
We have never told the user what the current working directory should be. We have not, before this PR, required a user to `git clone` this directory. This doc does not mention the need to `git clone`.

Reply:
@MikeSpreitzer do you prefer we invoke it using curl with url of the script?
doing that is a self reference isn’t it?

do you have a suggestion?

please keep in mind that this is (again) temp state until @francostellari completes the helm chart (which can be done only AFTER this PR is merged)

Reply:
The design here requires the user to get a copy of the repo, using either `git clone` or fetch an archive and unpack. Since it is required, it should be clearly said.

Unfortunately, I think this **does** introduce another self-reference --- if we follow our usual style. Many repos avoid literal self references here by instead telling the user to pick a release and then describing how to use the chosen release without literally including the release identifier in those instructions. We have been congratulating ourselves on having instructions that can be sped through with just a few copy-and-pasted lines, which is not quite consistent with that. I am not opposed to switching to that other style, but I think it should be done with a larger consensus than just two people. I know @pdettori also had short copy-able instructions for KSLight. I know that @clubanderson has opinions.

Reply:
so far we have tried to avoid requiring users to clone the repo. It adds some burden to users that just want to quickly try to run the examples. I understand that the helm chart would come later, but I am wondering if in the meantime we could do something like curl the raw script from GitHub and pipe it into sh. If this is not acceptable for some folks or too complex at this stage I am not opposed to move ahead with the current approach but with the understanding that we after this release is out we should really focus on improving UX, that is, publish a helm chart and use that (perhaps in addition to pch) as the only method of installation for a release.

Reply:
and it keeps the pattern of a copy paste simple step

Reply:
I can live with curl-to-bash too.
If this PR introduces another self-reference then it should include an update to release.md.

Reply:
ok. I actually understand now that's a bit problematic.
this file doesn't exist yet in git (only in my fork) so I can't put curl to to bash pointing to this file.
but ideally I would like to do that.
@MikeSpreitzer is it ok to change this in a followup PR?

Reply:
after we merge this PR, the file will exist..

Reply:
I think you could make all the changes in this PR. As you say, once this PR merges, a file that it introduces will exist. We do not test examples.md in CI.